packaging/nnstreamer.spec

```diff
@@ -14,6 +14,7 @@
 %define		gstpostfix	gstreamer-1.0
 %define		gstlibdir	%{_libdir}/%{gstpostfix}
 %define		nnstbindir	lib/nnstreamer/bin
+%define		builddir	build
```

Comment:
If 'build' is a directory commonly used by meson build, I wonder if it is okay to hardcoding it without defining it.
Thank you.